Algorithm/QuantConnect.Algorithm.csproj

```diff
@@ -72,6 +72,10 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\NodaTime.1.3.1\lib\net35-Client\NodaTime.dll</HintPath>
     </Reference>
+    <Reference Include="Python.Runtime, Version=4.0.0.1, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\AlexCatarino.PythonNET.Runtime.1.0.9\lib\UCS2\Python.Runtime.dll</HintPath>
```

Comment:
I created this nuget package with my personal account in nuget. I think QC should create its own package in its own account.